tools/run_tests/artifact_targets.py

```diff
@@ -226,6 +226,25 @@ def build_jobspec(self):
                                self.gyp_arch])
 
 
+class PHPExtArtifact:
+  """Builds PHP native extension"""
+
+  def __init__(self, platform, arch):
+    self.name = 'php_ext_{0}_{1}'.format(platform, arch)
```

Comment:
nit: this builds also the pecl  package itself, so the PHPExt name is not really accurate.